host/xtest/Makefile

```diff
@@ -120,6 +120,9 @@ endif
 ifeq ($(CFG_RPMB_FS),y)
 CFLAGS += -DCFG_RPMB_FS
 endif
+ifeq ($(CFG_SQL_FS),y)
```

Comment:
Good idea. Unfortunately, currently it causes a compile error in `xtest_7000_gp.c`:
```
  CC      /home/jerome/work/hikey_optee/optee_test/host/xtest/xtest_7000_gp.c
In file included from /home/jerome/work/hikey_optee/optee_os/out/arm-plat-hikey/export-ta_arm32/host_include/tee_internal_api.h:32:0,
                 from /home/jerome/work/hikey_optee/optee_test/host/xtest/xtest_7000_gp.c:27:
/home/jerome/work/hikey_optee/optee_os/out/arm-plat-hikey/export-ta_arm32/host_include/tee_api.h:37:19: fatal error: trace.h: No such file or directory
 #include <trace.h>
                   ^
compilation terminated.
Makefile:139: recipe for target '/home/jerome/work/hikey_optee/optee_test/out/xtest/xtest_7000_gp.o' failed
```

I see no reason why `xtest_*.c` should include any TA header (`tee_internal_api.h`), so I guess it's a bug with how we integrate the GP test suite in `xtest`.

Can we keep this for a separate commit? I'll put it on my to-do list.